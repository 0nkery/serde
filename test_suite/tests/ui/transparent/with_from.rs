#[macro_use]
extern crate serde_derive;

#[derive(Serialize)]
#[serde(transparent, from = "u64")]
struct S {
    a: u8,
}

fn main() {}
