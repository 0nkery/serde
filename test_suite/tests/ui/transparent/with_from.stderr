error: #[serde(transparent)] is not allowed with #[serde(from = "...")]
 --> $DIR/with_from.rs:5:1
  |
5 | / #[serde(transparent, from = "u64")]
6 | | struct S {
7 | |     a: u8,
8 | | }
  | |_^

error: aborting due to previous error

